Update test rapport (#43)
2nd-factor/docs: TestRapport.md

### Inleiding
Voor het behalen van verschillenden testen is het bepalen van de strategie de eerste stap. Hierbij kan gekozen worden uit een brede variatie aan strategieën.  
Er zijn een aantal populaire strategieën die je kunt gebruiken als leidraad. Zoals Big-bang, Top-down, Bottom-up, Sandwich en Modified Sandwich.

### Strategie
Voor 2nd-Factor hebben we gekozen voor bottom-up. Dit hebben we gekozen omdat sommige individuele componenten eerder klaar waren dan andere. 

### Conclusie

De testen doorgenomen als beschreven. De testen zijn gedaan in de volgorde zoals het V-model loopt. De Integratietesting zijn geautomatiseerd. 

> Met deze geautomatiseerd code kan je de front en hardware mokken.

Reviewer: ```suggestion
Met deze geautomatiseerde* code kun* je de front-* en hardware mocken*.
```

> Met deze geautomatiseerde code kun je de front- en hardware mocken.

Reviewer: ```suggestion
Met deze geautomatiseerde code kun je de front-end en hardware mocken.
```

Sorry, ik had deze zin verkeerd gelezen

> Het System Test plan is met meerdere personen uitgevoerd. Daarbij hebben twee testen een andere resultaat dan verwacht maar in positief zin.

Reviewer: ```suggestion
Het System Test plan is met meerdere personen uitgevoerd. Daarbij hebben twee testen een andere resultaat dan verwacht maar in positieve* zin.
```

> De laatste test is de User Acceptance Test. Deze is ook met meerdere uitgevoerd. Ook deze test is gelsaagd. Daarbij zijn er twee testen(Test AD-power_01) en (Test AD-power_00) die een andere resultaat leveren dan verwacht. Maar ook in positief zin.

Reviewer: ```suggestion
De laatste test is de User Acceptance Test. Deze is ook met meerdere personen* uitgevoerd. Ook deze test is geslaagd*. Daarbij zijn er twee testen(Test AD-power_01) en (Test AD-power_00) die een andere resultaat leveren dan verwacht. Maar ook in positieve* zin.
```

> Zie de bijlagen voor alle de testen.

Reviewer: ```suggestion
Zie de bijlagen voor alle testen.
```

Reviewer 2: die laatste lijkt me niet juist
